draft-fossati-tls-attestation.md

```diff
@@ -1061,6 +1075,22 @@ the TLS server will terminate the exchange.
 
 TBD.
 
+# Privacy Considerations {#priv-cons}
+
+In background-check mode, the Verifier has access to detailed information about the Attester's TCB through Evidence.
+The Verifier also knows the exact time and the party with whom the secure channel establishment is attempted (i.e., the RP).
+The privacy implications are similar to online OCSP {{-ocsp}}.
+Although the Verifier is trusted by the RP, the same may not apply to the Attester.
```

Comment:
The meaning of the word "trusted" here is too broad. The RP is trusted to provide some services to the client, but not necessarily trusted with the client's private information. Obviously the trust situation with the Verifier is different.